config/routes.rb

```diff
@@ -18,7 +18,7 @@
   namespace :insights_cloud do
     resources :tasks, only: [:create]
     resource :settings, only: [:show, :update]
-    resources :hits, except: %i[show] do
+    resources :hits, only: [:index, :show] do
```

Comment:
```suggestion
    resources :hits, only: [:index] do
```

with `show` included there is an issue in the old host page: `ActionController::ParameterMissing in InsightsCloud::HitsController#show`